.gitattributes

```diff
@@ -1,3 +1,4 @@
+## Ahmad's test ##
```

Comment:
Sure thanks for doing this exercise with me, Jerome!

Best regards,
Ahmad Fattahi, PhD | Group Leader, Development Support <vcampus.osisoft.com> | OSIsoft, LLC, 777 Davis St, San Leandro, CA

From: Jerome Lefebvre [mailto:notifications@github.com]
Sent: Sunday, January 25, 2015 9:32 PM
To: JeromeLefebvre/abetterpie
Cc: Ahmad Fattahi
Subject: Re: [abetterpie] Update .gitattributes (#12)


In .gitattributes<https://github.com/JeromeLefebvre/abetterpie/pull/12#discussion_r23513259>:

> @@ -1,3 +1,4 @@

> +## Ahmad's test ##

Sorry that I had to delete this line.

—
Reply to this email directly or view it on GitHub<https://github.com/JeromeLefebvre/abetterpie/pull/12/files#r23513259>.